bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java

```diff
@@ -7,6 +7,7 @@
  */
 package org.eclipse.smarthome.model.script.actions;
 
+import org.quartz.utils.FindbugsSuppressWarnings;
```

Comment:
[edu.umd.cs.findbugs.annotations.SuppressFBWarnings](http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/SuppressFBWarnings.html) would be the best choice, but it is not part of the Target Platform at the moment.

I was also surprised that the code above is working, but it is.